gbpservice/neutron/tests/unit/nfp/configurator/test_data/fw_test_data.py

```diff
@@ -28,9 +28,9 @@ class FakeObjects(object):
     drivers = 'drivers'
     vm_mgmt_ip = '172.24.4.5'
     service_vendor = 'service_vendor'
-    source_cidrs = ['1.2.3.4/24']
+    source_cidrs = ['1/24']
     destination_cidr = 'destination_cidr'
-    gateway_ip = '1.2.3.4'
```

Comment:
Why is the test data modified? That too with incorrect IPs, MACs and other data?